core/src/main/java/com/google/cloud/sql/core/Refresher.java

```diff
@@ -32,7 +32,7 @@
 
 /** Handles periodic refresh operations for an instance. */
 class Refresher {
-  private static final Logger logger = Logger.getLogger(CloudSqlInstance.class.getName());
+  private static final Logger logger = Logger.getLogger(DefaultConnectionInfoCache.class.getName());
```

Comment:
it should use `Refresher` class instead of `DefaultConnectionInfoCache`

Reply:
Fixed.